ITEM_en.tsv

```diff
@@ -78,7 +78,7 @@ ITEM_20150317_000088	An improved version of the Gladius by extending its blade l
 ITEM_20150317_000089	Aras' Falchion
 ITEM_20150317_000090	This weapon gives you the opportunity to experience the quality of the Falchion.
 ITEM_20150317_000091	Smith Sword
-ITEM_20150317_000092	- 2% chance to cause an additional +600% damage.
+ITEM_20150317_000092	2% chance to cause an additional +600% damage.
```

Comment:
I think that "-" it's just a wrong separator and not a minus because in the korean text it would be added near 2%